boxes/identity-server-analytics/confs/repository/conf/registry.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<!--
+  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<wso2registry>
+
+    <!--
+    For details on configuring different config & governance registries see;
+    http://wso2.org/library/tutorials/2010/04/sharing-registry-space-across-multiple-product-instances
+    -->
+
+    <currentDBConfig>wso2registry</currentDBConfig>
+    <readOnly>false</readOnly>
+    <enableCache>true</enableCache>
+    <registryRoot>/</registryRoot>
+
+
+    <dbConfig name="govregistry">
+        <dataSource>jdbc/WSO2REG_DB</dataSource>
+    </dbConfig>
+    <remoteInstance url="https://localhost:9443/registry">
+        <id>gov</id>
+        <cacheId>root@jdbc:mysql://172.28.128.3:3306/wso2isanalytics</cacheId>
+        <dbConfig>govregistry</dbConfig>
+        <readOnly>false</readOnly>
+        <enableCache>true</enableCache>
+        <registryRoot>/</registryRoot>
+    </remoteInstance>
+    <mount path="/_system/governance" overwrite="true">
```

Comment:
Do we share any registry configs in between IS and analytics ? I believe that we only need the local registry config, WDYT?

Reply:
@DilanUA I have fixed it.